CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ We appreciate your patience while we speedily work towards a stable release of t
 
 <!-- NEW CONTENT GENERATED BELOW. PLEASE PRESERVE THIS COMMENT. -->
 
+### 0.64.227 (2024-10-25)
+
+- The `modal container list` CLI command now only shows containers for your active environment. You can pass `--env` to list containers in other environments.
```

Comment:
"Active environment" is maybe slightly inaccurate (though in a very common and confusing way) — we don't technically have this concept. We have an active _profile_, and a profile can have an environment configuration, but it doesn't have to, in which case the server determines the environment to use. But I'm not certain how better to articulate it. We should maybe rethink how this works.

Reply:
😅 yeah just switching environments was confusing when trying this out too, definitely something we should rethink!

Updated the changelog to better match how it works though, thanks!